ConfigSpace/forbidden.py

```diff
@@ -29,14 +29,16 @@
 from abc import ABCMeta, abstractmethod
 import operator
 
-import six
+import io
+from functools import reduce
 
-from ConfigSpace.hyperparameters import Hyperparameter
 
+from ConfigSpace.hyperparameters import Hyperparameter
+from typing import List, Dict, Any, Union
 
 class AbstractForbiddenComponent(object):
     __metaclass__ = ABCMeta
-
+    # todo : should absract methods be annotated with types?
```

Comment:
From my point of view, this is not necessary here. Did mypy complain? If yes, we have to do something about it. Else, please remove the comment. The other abstract methods which should return something should be annotated though.